src/condor_startd.V6/util.cpp

```diff
@@ -323,6 +323,29 @@ bool retry_cleanup_execute_dir(const std::string & path, int /*options*/, int &
 	return success;
 }
 
+bool
+retry_cleanup_logical_volume(const std::string& lv_name, int options, int& err) {
+	auto * volman = resmgr->getVolumeManager();
+	ASSERT(volman);
+	ASSERT(volman->is_enabled());
+
+	bool success = true;
+
+	// Attempt LV cleanup
+	CondorError error;
+	int status = volman->CleanupLV(lv_name, error, options);
+	if (status && status != 2) {
+		dprintf(D_FULLDEBUG, "Failed to cleanup LV %s: %s\n", lv_name.c_str(), error.getFullText().c_str());
+		success = false;
+		err = status;
+	} else {
+		dprintf(D_FULLDEBUG, "Successfully cleaned up LV: %s\n", lv_name.c_str());
```

Comment:
might be nice to log that we didn't actually clean up the LV so much as notice it was gone.